pandas/tslib.pyx

```diff
@@ -895,7 +900,7 @@ _implemented_methods.extend(_nan_methods)
 
 for _method_name in _nat_methods:
     # not all methods exist in all versions of Python
-    if hasattr(NaTType, _method_name):
+    if hasattr(NaTType, _method_name) or _method_name in _round_methods:
```

Comment:
@jreback `NaTType` class doesn't has a `round` method. It would unnecessary if I'll define `round` abstract method at the `_Timestamp` class.